pages/1.13/security/ent/tls-ssl/exhibitor/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle:  Securing Exhibitor with mutual TLS
+title: Securing Exhibitor with mutual TLS
+menuWeight: 500
+excerpt: Securing DC/OS with a TLS enabled Exhibitor ensemble
+enterprise: true
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+By default, the Exhibitor HTTP service is open to any client that can reach port 8181 on a master node. This documentation chapter describes a method for protecting the Exhibitor service from unauthorized access. Once enabled, HTTP clients must access Exhibitor through Admin Router; thus applying the Admin Router access control policy to the exhibitor service.
+
+*Note: When accessing Exhibitor through Admin Router (**https://master_host/exhibitor**), authenticated users must have the `dcos:adminrouter:ops:exhibitor` privilege*
+
+# Securing Exhibitor
+
+The strategy for securing exhibitor is mutual TLS authentication. In order to secure exhibitor we must first create a unique root CA. This CA must be used to sign a client certificate for each master node. Creating a public key infrastructure that outputs PEM and Java Keytool formatted artifacts is not a trivial task. To make this processes easier, a simple tool has been created for producing the necessary files. 
+
+*Note: This guide assumes a static exhibitor ensemble. Dynamic masters are not currently supported*
```

Comment:
Our users don't necessarily know what this means. Let's describe this in terms of DC/OS config, please.

Reply:
It's probably a good opportunity for _us_ to find out what this _exactly_ means :D  

Relevant config options and documentation pieces:


https://docs.mesosphere.com/1.12/installing/production/advanced-configuration/configuration-reference/#exhibitor-storage-backend

NOTE: If master_discovery is set to `master_http_loadbalancer`, then exhibitor_storage_backend cannot be set to `static`.

`master_discovery: static`

...